### DISCUSSION TITLE
Is there way to prevent access to the listing of file in filer directory?

### DISCUSSION THREAD

**Discussion by ryanchen34057** [seaweedfs/seaweedfs#1766] [Q&A]
Hi first off, thank you for this amazing project!
Just a quick question, when I set up my filer server, I noticed if I uploaded some files under a directory, I can access to that directory and see some amount of files under it.
I'm looking for a way to prevent this because in our use case(image uploading service), there's no point showing users this information.
Is there any way to do this? Thank you in advance.

**Comment by chrislusf** [comment]
Thanks! See 
```
$ weed filer -h
Example: weed filer -port=8888 -master=<ip:port>[,<ip:port>]*
Default Usage:
  ...
  -dirListLimit int
    	limit sub dir listing size (default 100000)
  -disableDirListing
    	turn off directory listing
  ...
```

**Reply by ryanchen34057** [reply]
Totally missed that! 
Thank you for responding.
